nowcasting_dataset/data_sources/gsp/gsp_data_source.py

```diff
@@ -436,6 +441,10 @@ def load_solar_gsp_data(
     gsp_capacity_df = gsp_capacity_df.dropna(axis="columns", how="all")
     gsp_capacity_df = gsp_capacity_df.clip(lower=0, upper=5e7)
 
+    # remove how rows of nans
```

Comment:
Should the "how" be removed from this comment?

Reply:
"whole"